uat/src/main/resources/greengrass/features/log-manager-1.feature

```diff
@@ -72,26 +74,28 @@ Feature: Greengrass V2 LogManager
         }
         """
         And I deploy the Greengrass deployment configuration
-        Then the Greengrass deployment is COMPLETED on the device after 3 minutes
+        Then the Greengrass deployment is COMPLETED on the device after 5 minutes
         Then I verify the aws.greengrass.LogManager component is RUNNING using the greengrass-cli
-        Then I verify that it created a log group for component type GreengrassSystemComponent for component System, with streams within 120 seconds in CloudWatch
         And I verify that it created a log group for component type UserComponent for component UserComponentA, with streams within 120 seconds in CloudWatch
+        And I verify the rotated files are deleted and that the active log file is present for component UserComponentA
 
-    @smoke
-    Scenario: LogManager-1-T2: As a customer I can configure the logs uploader to delete log files after all logs from the file have been uploaded to CloudWatch
+    @R1    @functional @M2 @B1 @stable
+    Scenario: LogManager-1-T3: As a customer I can configure the logs uploader to delete log oldest log files inorder to keep the disk space limit configured by the customer
+        Given 10 temporary rotated log files for component UserComponentB have been created
         Given I create a Greengrass deployment with components
             | aws.greengrass.Cli        | LATEST |
             | aws.greengrass.LogManager | LATEST |
-        And I update my Greengrass deployment configuration, setting the component aws.greengrass.LogManager configuration to:
-         """
+        When I update my Greengrass deployment configuration, setting the component aws.greengrass.LogManager configuration to:
```

Comment:
All this steps are mixed up now please fix them t1-b is gone and I can't tell which change is which from the changes